test/utils/networkpolicy_helper.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2023 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+
+	"antrea.io/nephe/pkg/labels"
+	k8stemplates "antrea.io/nephe/test/templates"
+)
+
+// ConfigANPApplyTo helper function to configure appliedTo in Antrea NetworkPolicy.
+func ConfigANPApplyTo(kind, instanceName, vpc, tagKey, tagVal string) *k8stemplates.EntitySelectorParameters {
```

Comment:
Why splitting these two func into a separate file? Following this logic should we also split other funcs in np e2e tests like verifyXXX into this file also? This way upgrade test can also reuse some helper funcs. Personally I'm not a fan of current inline funcs, since they are a bit hard to track and IDEs doesn't highlight them. No need to be done in this PR though.

Reply:
This was used as is in both files, so I placed them in common. This is config, should be place. Verify I didnt put, because integration NP test have can more detailed verification step, then upgrade. Wanted to keep upgrade test simple. Note: This is not the final steps in upgrade. We will add more tests in upgrade tests.